src/Listener/JsonApiListener.php

```diff
@@ -483,6 +483,12 @@ protected function _getEntityList($entityName, $associations)
      */
     protected function _checkRequestData()
     {
+        // prevent false positives for GET requests using query parameters
+        // as those will also have request data
```

Comment:
You don't need to update comment but simply remove handling of this case. It's the dev's job to disable the querystring to data setting feature of Prg component as doing so makes no sense for an API response.